package.json

```diff
@@ -73,8 +73,7 @@
     "pdfjs-dist": "^2.5.207",
     "postcss": "8.4.32",
     "postcss-loader": "5.3.0",
-    "puppeteer": "npm:puppeteer-core@^22.15.0",
-    "puppeteer-core": "^21.3.8",
```

Comment:
I think we still need to use core as we don't need a bundled browser from the main package